src/infra/metrics/metrics_types.rs

```diff
@@ -85,6 +86,15 @@ impl From<String> for MetricLabelValue {
     }
 }
 
+impl From<Option<Address>> for MetricLabelValue {
+    fn from(value: Option<Address>) -> Self {
+        match value {
```

Comment:
suggestion:
change this entire match to `value.map(|x| x.to_string())`

same behaviour, smaller code

Reply:
Thanks for suggestions, I've found that it works also with this implementation:

```
impl From<Option<Address>> for MetricLabelValue {
    fn from(value: Option<Address>) -> Self {
        Self::Some(value.or_empty())
    }
}
```

but I've not committed, because I saw implementation with `match` for `From<Option<&str>>` and decided leave it as is yet

Reply:
Just saw the Self is manually defining Some and None, so you can leave it as it was, with the match. No need to change anything.